src/script/campaign/mod/vco-listeners-nor.lua

```diff
@@ -96,7 +96,6 @@ local function add_listeners()
 		"FactionTurnStart",
 		function(context)
 			return cm:model():turn_number() == 1 and
-				context:faction():is_human() and
```

Comment:
This has to be done for all the others that unlock units. Kislev and Dark Elves as far as I know.